addons/website_sale/controllers/main.py

```diff
@@ -386,6 +375,18 @@ def shop(self, page=0, category=None, search='', min_price=0.0, max_price=0.0, p
         fiscal_position_sudo = website.fiscal_position_id.sudo()
         products_prices = lazy(lambda: products._get_sales_prices(pricelist, fiscal_position_sudo))
 
+        attributes_values = lazy(lambda: request.env['product.attribute.value'].browse(attrib_set))
```

Comment:
is this lazy really changing anything here?
I feel like its evaluation will be forced on next line anyway :thinking: